{"name": "test_query_existinggebouwen", "status": "broken", "statusDetails": {"message": "requests.exceptions.MissingSchema: Invalid URL '/v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3': No schema supplied. Perhaps you meant http:///v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3?", "trace": "auth_header = {'Authorization': 'Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJXNTRsQzhsei1Yek9tVDhOMDFCTXdXLXlYbTFodGpH...oLqfwgaPwt8FhjblrBpvqv52jxFx2UmrFYG4_TKgJOlmUrFxvcvoRw2SdW_iVkemCV8tfCk918__OE01EDpmL09YiU1SzbQ8XsfpM5EHgroDSGxgcs8zA'}\n\n    @allure.step(\"API Call `https://acc.api.data.amsterdam.nl/v1/standvastgoed/gebouwen/` to check the existing property\")\n    def test_query_existinggebouwen(auth_header):\n    \n        \"\"\"Prove that query-ing by `pndId` finds property.\"\"\"\n    \n        payload = {\"id\": '2022-01-01.0363200000205917.3'}\n        payload1 = {\"naam\": \"Molenpad\"}\n>       response = requests.get(dataconstantstandvastgoed.payloadparams, params=payload1, headers=auth_header)\n\ne2etest/test_datasetstandvastgoed.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n../venv/lib/python3.8/site-packages/requests/api.py:76: in get\n    return request('get', url, params=params, **kwargs)\n../venv/lib/python3.8/site-packages/requests/api.py:61: in request\n    return session.request(method=method, url=url, **kwargs)\n../venv/lib/python3.8/site-packages/requests/sessions.py:528: in request\n    prep = self.prepare_request(req)\n../venv/lib/python3.8/site-packages/requests/sessions.py:456: in prepare_request\n    p.prepare(\n../venv/lib/python3.8/site-packages/requests/models.py:316: in prepare\n    self.prepare_url(url, params)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <PreparedRequest [GET]>, url = '/v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3', params = OrderedDict([('naam', 'Molenpad')])\n\n    def prepare_url(self, url, params):\n        \"\"\"Prepares the given HTTP URL.\"\"\"\n        #: Accept objects that have string representations.\n        #: We're unable to blindly call unicode/str functions\n        #: as this will include the bytestring indicator (b'')\n        #: on python 3.x.\n        #: https://github.com/psf/requests/pull/2238\n        if isinstance(url, bytes):\n            url = url.decode('utf8')\n        else:\n            url = unicode(url) if is_py2 else str(url)\n    \n        # Remove leading whitespaces from url\n        url = url.lstrip()\n    \n        # Don't do any URL preparation for non-HTTP schemes like `mailto`,\n        # `data` etc to work around exceptions from `url_parse`, which\n        # handles RFC 3986 only.\n        if ':' in url and not url.lower().startswith('http'):\n            self.url = url\n            return\n    \n        # Support for unicode domain names and paths.\n        try:\n            scheme, auth, host, port, path, query, fragment = parse_url(url)\n        except LocationParseError as e:\n            raise InvalidURL(*e.args)\n    \n        if not scheme:\n            error = (\"Invalid URL {0!r}: No schema supplied. Perhaps you meant http://{0}?\")\n            error = error.format(to_native_string(url, 'utf8'))\n    \n>           raise MissingSchema(error)\nE           requests.exceptions.MissingSchema: Invalid URL '/v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3': No schema supplied. Perhaps you meant http:///v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3?\n\n../venv/lib/python3.8/site-packages/requests/models.py:390: MissingSchema"}, "description": "Prove that query-ing by `pndId` finds property.", "steps": [{"name": "API Call `https://acc.api.data.amsterdam.nl/v1/standvastgoed/gebouwen/` to check the existing property", "status": "broken", "statusDetails": {"message": "requests.exceptions.MissingSchema: Invalid URL '/v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3': No schema supplied. Perhaps you meant http:///v1/standvastgoed/gebouwen/?nummeraanduidingVolgnummer=3?\n", "trace": "  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/Users/jeswaran/projects/dso-api/src/e2etest/test_datasetstandvastgoed.py\", line 37, in test_query_existinggebouwen\n    response = requests.get(dataconstantstandvastgoed.payloadparams, params=payload1, headers=auth_header)\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/api.py\", line 76, in get\n    return request('get', url, params=params, **kwargs)\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/api.py\", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/sessions.py\", line 528, in request\n    prep = self.prepare_request(req)\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/sessions.py\", line 456, in prepare_request\n    p.prepare(\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/models.py\", line 316, in prepare\n    self.prepare_url(url, params)\n  File \"/Users/jeswaran/projects/dso-api/venv/lib/python3.8/site-packages/requests/models.py\", line 390, in prepare_url\n    raise MissingSchema(error)\n"}, "parameters": [{"name": "auth_header", "value": "{'Authorization': 'Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJXNTRsQzhsei1Yek9tVDhOMDFCTXdXLXlYbTFodGpHRlBRR0FUQm1SNU9BIn0.eyJleHAiOjE2NDY5MDIwMDQsImlhdCI6MTY0NjkwMTcwNCwianRpIjoiODRkZWRmYzMtMGY4NC00MGM3LWE4YWMtMThlMjZlYTk5ZTRkIiwiaXNzIjoiaHR0cHM6Ly9pYW0uYW1zdGVyZGFtLm5sL2F1dGgvcmVhbG1zL2RhdGFwdW50LWFkLWFjYyIsInN1YiI6IjEyYjc0YzZmLTRhNmUtNGFmMi05NzUzLTk1Yjg0MjdjMDE4MCIsInR5cCI6IkJlYXJlciIsImF6cCI6ImRhdGFkaWVuc3Rlbi10ZXN0IiwiYWNyIjoiMSIsInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJmcF9zdHVyaW5nc21pZGRlbGVuIiwiZnBfbWR3Il19LCJzY29wZSI6InByb2ZpbGUgZW1haWwiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsImNsaWVudEhvc3QiOiIxMC4yNDMuMjQuMTIiLCJjbGllbnRJZCI6ImRhdGFkaWVuc3Rlbi10ZXN0IiwicHJlZmVycmVkX3VzZXJuYW1lIjoic2VydmljZS1hY2NvdW50LWRhdGFkaWVuc3Rlbi10ZXN0IiwiY2xpZW50QWRkcmVzcyI6IjEwLjI0My4yNC4xMiJ9.UC9mB_THzN7f65WgXTID-c2OCJ0bENZzcpIfSzQ5JsDU5hHkJqrBSsUrb_Khxhu7Sv33X9yfHpSEFowHwRwbA5RJ4P5_gGaovZjEc3LZOaV0T8T7q4wQNd3K-DOSu8YDWXp6TZxwVviDbGDpEtF-QJ6x2Ho5dorFgIT2smKpK7tOgI6dKeMawede7w-2QModbe6Lq6FEq_Q8fdZf5jGq8dRxN9Wzby2groLqfwgaPwt8FhjblrBpvqv52jxFx2UmrFYG4_TKgJOlmUrFxvcvoRw2SdW_iVkemCV8tfCk918__OE01EDpmL09YiU1SzbQ8XsfpM5EHgroDSGxgcs8zA'}"}], "start": 1646901717151, "stop": 1646901717152}], "start": 1646901717151, "stop": 1646901717177, "uuid": "63d46907-9cce-46aa-984a-adbf14520140", "historyId": "ec4cd297469c8978d24614163c24333e", "testCaseId": "779d204f8cd9cc42eb8209a00bafb6a6", "fullName": "e2etest.test_datasetstandvastgoed#test_query_existinggebouwen", "labels": [{"name": "parentSuite", "value": "e2etest"}, {"name": "suite", "value": "test_datasetstandvastgoed"}, {"name": "host", "value": "Jayashrees-MacBook-Pro.local"}, {"name": "thread", "value": "63404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2etest.test_datasetstandvastgoed"}]}